test/utils/image/manifest.go

```diff
@@ -48,51 +48,45 @@ func (i *ImageConfig) SetVersion(version string) {
-	APIServer                = ImageConfig{e2eRegistry, "k8s-aggregator-sample-apiserver", "1.7v2", true}
+	AdmissionWebhook         = ImageConfig{e2eRegistry, "webhook", "1.10v2", false}
+	APIServer                = ImageConfig{e2eRegistry, "sample-apiserver", "1.0", false}
 	AppArmorLoader           = ImageConfig{gcRegistry, "apparmor-loader", "0.1", false}
 	BusyBox                  = ImageConfig{gcRegistry, "busybox", "1.24", false}
 	CheckMetadataConcealment = ImageConfig{gcRegistry, "check-metadata-concealment", "v0.0.3", false}
-	ClusterTester            = ImageConfig{e2eRegistry, "clusterapi-tester", "1.0", true}
-	CudaVectorAdd            = ImageConfig{e2eRegistry, "cuda-vector-add", "1.0", true}
-	Dnsutils                 = ImageConfig{e2eRegistry, "dnsutils", "1.0", true}
-	DNSMasq                  = ImageConfig{gcRegistry, "k8s-dns-dnsmasq", "1.14.5", true}
+	CudaVectorAdd            = ImageConfig{e2eRegistry, "cuda-vector-add", "1.0", false}
+	Dnsutils                 = ImageConfig{e2eRegistry, "dnsutils", "1.1", false}
 	EchoServer               = ImageConfig{gcRegistry, "echoserver", "1.10", false}
-	EntrypointTester         = ImageConfig{e2eRegistry, "entrypoint-tester", "1.0", true}
-	E2ENet                   = ImageConfig{gcRegistry, "e2e-net", "1.0", true}
-	Fakegitserver            = ImageConfig{e2eRegistry, "fakegitserver", "1.0", true}
-	GBFrontend               = ImageConfig{sampleRegistry, "gb-frontend", "v5", true}
-	GBRedisSlave             = ImageConfig{sampleRegistry, "gb-redisslave", "v2", true}
-	Goproxy                  = ImageConfig{e2eRegistry, "goproxy", "1.0", true}
-	Hostexec                 = ImageConfig{e2eRegistry, "hostexec", "1.1", true}
-	IpcUtils                 = ImageConfig{e2eRegistry, "ipc-utils", "1.0", true}
-	Iperf                    = ImageConfig{e2eRegistry, "iperf", "1.0", true}
-	JessieDnsutils           = ImageConfig{e2eRegistry, "jessie-dnsutils", "1.0", true}
-	Kitten                   = ImageConfig{e2eRegistry, "kitten", "1.0", true}
-	Liveness                 = ImageConfig{e2eRegistry, "liveness", "1.0", true}
-	LogsGenerator            = ImageConfig{e2eRegistry, "logs-generator", "1.0", true}
-	Mounttest                = ImageConfig{e2eRegistry, "mounttest", "1.0", true}
-	MounttestUser            = ImageConfig{e2eRegistry, "mounttest-user", "1.0", true}
-	Nautilus                 = ImageConfig{e2eRegistry, "nautilus", "1.0", true}
-	Net                      = ImageConfig{e2eRegistry, "net", "1.0", true}
-	Netexec                  = ImageConfig{e2eRegistry, "netexec", "1.0", true}
-	Nettest                  = ImageConfig{e2eRegistry, "nettest", "1.0", true}
+	EntrypointTester         = ImageConfig{e2eRegistry, "entrypoint-tester", "1.0", false}
+	Fakegitserver            = ImageConfig{e2eRegistry, "fakegitserver", "1.0", false}
+	GBFrontend               = ImageConfig{sampleRegistry, "gb-frontend", "v5", false}
```

Comment:
it doesn't look like the gb- images are using manifest lists yet, so setting `false` here is probably wrong.

Reply:
Hmm, i eyeballed it here - https://console.cloud.google.com/gcr/images/google-samples/GLOBAL (searched there for `gb-`) let me check if something is wrong by building those images locally

Reply:
you are right. i missed that. assumed that https://github.com/kubernetes/examples/commit/5f484579b5cc04900591c61cf87ece00fe8426d7 and https://github.com/kubernetes/examples/commit/5f484579b5cc04900591c61cf87ece00fe8426d7 was doing it, looks like it's not.